static/js/main.js

```diff
@@ -25,6 +25,33 @@ document.addEventListener("DOMContentLoaded", function () {
       console.error(error);
     });
 
+
+    // Flash message success 
+  const button = document.getElementById("main-call-button");
+  const successAlertPlaceholder = document.getElementById("success-alert-placeholder");
+
+  button.addEventListener("click", function () {
+    // Success alert
+    const successAlert = document.createElement("div");
+    successAlert.classList.add("alert", "alert-success");
+    successAlert.textContent = "Images have been refreshed successfully!";
+
+    successAlertPlaceholder.innerHTML = "";
+    successAlertPlaceholder.appendChild(successAlert);
+
+    // Timer set to 5 seconds
+    setTimeout(function () {
```

Comment:
try to find a bootstrap way with animation and automatically dismissible alerts https://getbootstrap.com/docs/5.0/components/alerts/#dismissing